server/events/working_dir.go

```diff
@@ -83,9 +82,8 @@ type FileWorkspace struct {
 func (w *FileWorkspace) Clone(
 	log logging.SimpleLogging,
 	headRepo models.Repo,
-	p models.PullRequest,
-	workspace string) (string, bool, error) {
-	cloneDir := w.cloneDir(p.BaseRepo, p, workspace)
+	p models.PullRequest) (string, bool, error) {
+	cloneDir := w.cloneDir(p.BaseRepo, p)
```

Comment:
This will always return the same directory for a pull request, this means that we can call `Clone()` how many times we want on the PR and it will make only one `git clone`.